src/common/TimeAgo/TimeAgo.tsx

```diff
@@ -51,13 +51,15 @@ export const TimeAgo = ({ date, ...rest }: TimeAgoProps) => {
     };
   }, [date, timestamp]);
 
+  const text = formatDistanceToNow(fromTS, { addSuffix: true }) === 'less than a minute ago' ? 'seconds ago' : formatDistanceToNow(fromTS, { addSuffix: true });

```

Comment:
This change should be done from within `formatDistanceToNow` and we should replace `less than a minute ago` with `seconds ago` for all cases.

Reply:
we've tried to find a better way to do this but he API is inflexible and limiting